app/views/layouts/application.html.erb

```diff
@@ -95,7 +95,7 @@
 
             <%= drop_down "Region Admin" do %>
               <%= menu_item "Donors", region_admin_donors_url %>
-              <%= menu_item "Recipients", recipients_locations_path %>
+              <%= menu_item "Recipients", region_admin_recipients_url %>
```

Comment:
The real change is adding the `region_admin` prefix.  I switched the `_path` to `_url` as a matter of preference